src/arena-beatem-up/Assets/Prefab/Arena/UI/Score.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class Score : MonoBehaviour
+{
+    public string ScorePrefix;
+    
+    private Text _scoreText;
+    private int _scoreAmount;
+
+    public static Score Instance { get; private set; }
+    private void Awake()
+    {
+        if (Instance != null && Instance != this)
```

Comment:
This will reset the instance whenever we create a new scene - I think it'd be best to keep it alive across scenes, especially if we want access to the score in the GameOver screen